src/app/tests/suites/TestAccessControlConstraints.yaml

```
@@ -28,21 +28,36 @@ tests:
               - name: "nodeId"
                 value: nodeId
 
+    - label: "Read the fabric ID from the alpha fabric"
```

```suggestion
    - label: "Read the fabric index from the alpha fabric"
```
But I am not sure we need this step at all; see below.

---

See https://github.com/project-chip/connectedhomeip/pull/29253#discussion_r1327094126